projects/plugins/boost/app/modules/image-guide/Image_Guide_Proxy.php

```diff
@@ -9,14 +9,13 @@ class Image_Guide_Proxy {
 	const NONCE_ACTION = 'jb-ig-proxy-nonce';
 
 	public static function init() {
-		$instance = new self();
-		add_action( 'wp_ajax_boost_proxy_ig', array( $instance, 'handle_ig_proxy' ) );
+		add_action( 'wp_ajax_boost_proxy_ig', 'Automattic\Jetpack_Boost\Modules\Image_Guide\Image_Guide_Proxy::handle_proxy' );
```

Comment:
Small note - I think this can be simplified further by doing
```php
add_action( 'wp_ajax_boost_proxy_ig' , array( __CLASS__, 'handle_proxy' ) );
```

That way, you won't need to update the namespace inline when we eventually change it.

Reply:
Thank you, that's useful!